vars/build.groovy

```diff
@@ -17,6 +18,11 @@ def getRepoUrl() {
   return sh(returnStdout: true, script: "git config --get remote.origin.url").trim()
 }
 
+// private
+def getRepoName(repoUrl) {
```

Comment:
I can't see where `repoUrl` is used, is it necessary?

Reply:
It's used when passing the status of the build to github